### DISCUSSION TITLE
Hiding interface elements

### DISCUSSION THREAD

**Discussion by danshearer** [Leantime/leantime#583] [General]
I have two quite different examples with a common desire: I would like to hide some UI elements:

1. Is it possible to hide the timesheet menu on the left hand side? References to timekeeping would I suppose remain the same in all other parts of the app, but for lightweight projects there is no need for timesheets at all. In fact, it can be off-putting for some team members. Leantime is good at creative support, and that often is needed most where timesheets are the least applicable.

2. On the task view, is it possible to hide columns? For the lightest-weight projects, at least "Awaiting Approval" can be hidden. Perhaps "Blocked" in some other lightest-weight cases.

Dan

**Comment by x2zdigital** [comment]
I think this or something like it would be a great idea, especially if it is role based.

**Comment by archlinux-fan** [comment]
+1 for having options for hiding things that are not in use.
I posted have a **[feature request](https://github.com/Leantime/leantime/issues/547)** a few days ago since I am using only a bare minimum version of leantime.

**Comment by marcelfolaron** [comment]
Thanks. In terms of architecture we should probably create a feature
service that can be used across the entire system. Features would be stored
as part of project settings.

Can call it in the controller of a module and pass it into the template.
